### ISSUE TITLE
Windows 10: NVDA does not announce when the system is ready to start logging after computer start.

### ISSUE THREAD

**Issue by MarioBatusic** [nvaccess/nvda#11222]
### Steps to reproduce:
1. Start your PC with Windows 10. 
  1. The only message from NVDA ist to wait for the system to start.
### Actual behavior:
NVDA does not announce the new system status: Ready for login.
### Expected behavior:
When the System is ready for logging in, NVDA should announce this.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.1
#### Windows version:
10.0.18363 Build 18363
#### Does the issue still occur after restarting your computer?
The issue occurs always, after each system start.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
It occures already from the beginning of Windows 10.
#### If addons are disabled, is your problem still occuring?
Yes.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by Adriani90** [comment]
Closing as duplicate of #10954.

**Comment by XLTechie** [comment]
@Adriani90, it may not be a duplicate.

This user doesn't say what he has to do to get speech back. The other issue 
seems to require re-selecting the synth to get speech back. It would be 
informative to know how the OP resolves this problem.

Furthermore, this user says it happens every time. I believe the other issue is 
intermittent.

It might be the same, but personally I'm not convinced.

**Comment by MarioBatusic** [comment]
### These Issues are not the same at allThe mentioned Issue is about basic problem with synth not working at all on the Logon window.
My Issue is about missing further Information - Status on the Logon Window. The only message said by NVDA is "Windows login. Please wait." Then after that I can wait till Doomsday. I don't get the Information at the time when it is possible to begin with login by pressing the Spacebar. That is my issue.

**Comment by XLTechie** [comment]
That's what I thought. Definitely different issues, and yours has existed for 
several NVDA versions.

Please re-open.

**Comment by Adriani90** [comment]
Technically it is the same underlying problem even though the description differs abit.

Von meinem iPhone gesendet

**Comment by XLTechie** [comment]
On Wed, 3 Jun 2020, Adriani90 wrote:

> Technically it is the same underlying problem even though the description differs abit.

I'm sorry, but you appear to have misunderstood something, because nobody 
understanding this issue could possibly make that claim.

In the issue you say this is a duplicate of, no synth is loaded. In order to get 
speech back, you have to Bring up the speech configuration, and select the synth 
again, all with no speech.

In this issue: after a system update, upon reboot, Windows says "Please wait, 
system updating" or something like that. Already it is completely different than 
the first issue, because there is speech.
Then after it finishes updating, and it is ready for login, it doesn't announce 
this fact, it just sits there doing nothing.
But if you press space, it talks fine and lets you log in.

How can these possibly be the same underlying problem?

In one there is no speech ever after startup but before login, until you play 
with the speech dialog of NVDA. In the other NVDA is simply failing to announce 
a state change, though speech is fine.

I agree that the broken English of the title and original report make it a bit 
more difficult to understand what is going on, but this is not a duplicate of 
the issue you sighted at all.

**Comment by XLTechie** [comment]
If I am wrong, and these are somehow the same issue, please sight some evidence 
of that from the code, one of the core devs, or some other relevant source.

The issue you linked is a much more serious synth not loading issue. This one 
starts with the synth being loaded, and failure to announce a change in system 
readiness, which is a UX issue.

**Comment by Adriani90** [comment]
See the comments in that issue and issue #6566

**Comment by josephsl** [comment]
Hi,

Does Narrator announce the new status? If yes, the issue cited has nothing to do with this issue and are two separate areas. I suspect event processing might be involved - that is, NVDA might be failing to detect a UIA event coming from logonui.exe.

Reopening. Thanks.

**Comment by josephsl** [comment]
Hi,

If #6566 has the same underlying problem as this one, then I propose that issue in favor of this unless information on this issue provides more helpful details.

Thanks.

**Comment by Adriani90** [comment]
Since no valuable input has been given on this issue so far, I am closing in favor of #6566.
